GNOME #331059; Ubuntu #79620; use /var/run for the mdm_socket file to comply
with the FHS and to avoid issues with bootclean; found in the Ubuntu package.

Index: mdm-2.18.1/daemon/mdm.h
===================================================================
--- mdm-2.18.1.orig/daemon/mdm.h	2007-04-09 07:15:15.000000000 +0200
+++ mdm-2.18.1/daemon/mdm.h	2007-05-02 17:54:03.000000000 +0200
@@ -798,7 +798,7 @@
  * all to be grabbed in one pull.
  */
 #define MDM_SUP_MAX_MESSAGES 80
-#define MDM_SUP_SOCKET "/tmp/.mdm_socket"
+#define MDM_SUP_SOCKET "/var/run/mdm_socket"
 
 /*
  * The user socket protocol.  Each command is given on a separate line
